problems/arraysWithLoops.js

```diff
@@ -83,7 +130,13 @@ function findSecondLargest() {}
  * @returns {array} nums without the duplicates
  */
 // Hint: Look into the `.includes` method.
-function removeDups() {}
+function removeDups(nums) {
+  let Dups = [];
+  for (let i = 0; i < nums.length; i++) {
+  if (! Dups.includes(nums[i])) {
+    Dups.push(nums[i])}
+  } return Dups
+}
```

Comment:
Nice ! Don't forget to properly format your code so that it's easier to read and debug